torq.q

```diff
@@ -574,20 +583,10 @@ sys:{[cmd]
 if[not `noconfig in key .proc.params;
 	// load TorQ Default configuration module
 	.proc.loadconfig[getenv[`KDBCONFIG],"/settings/";] each `default,.proc.parentproctype,.proc.proctype,.proc.procname;
```

Comment:
I think we should use the new function for this one as well, for consistency

Reply:
and have logic in the function to avoid the logging? I just figured it was going to be messier to include it in the function and would outweigh the consistency factor

Reply:
I think it's fine to do the logging for this one as well, makes it more consistent anyway